### DISCUSSION TITLE
[Projects Beta] Allow issue templates to automatically assign a project.

### DISCUSSION THREAD

**Discussion by gdavis** [community/community#13382] [Projects and Issues]
I would like the markdown format for creating new issues to automatically assign an issue to a project. This does not currently seem to be supported, and would save a lot of time reviewing issues and assigning them by doing it automatically.

My current workflow allows app users to create new issues that are assigned a label of "needs review" and I would like them to automatically move to a project where I have a view that filters labels so I can easily see new issues that need my attention.

**Comment by equinusocio** [comment]
This is a must have, we can assign tags, people, but not projects. Auto assignment is really important because issues without projects will not appear in the project board and cannot be tracked there.

**Reply by pierre-eliep-met** [reply]
agreed ! 💯

**Reply by Faolain** [reply]
Have you found a way around this @equinusocio ?

**Comment by Khamida-schemeverge** [comment]
Agreed! Its a must-have feature to save time on assigning the projects!

**Comment by danielracca** [comment]
Agreed!

**Comment by lukewar** [comment]
This is something we are looking into. In the meantime, if you are blocked, please consider using GitHub Actions: https://github.com/actions/add-to-project

**Comment by Johanmot** [comment]
> I would like the markdown format for creating new issues to automatically assign an issue to a project. This does not currently seem to be supported, and would save a lot of time reviewing issues and assigning them by doing it automatically.
> 
> My current workflow allows app users to create new issues that are assigned a label of "needs review" and I would like them to automatically move to a project where I have a view that filters labels so I can easily see new issues that need my attention.

**Comment by Faolain** [comment]
Why hasn't this been addressed yet? It's been a few years, I can only think that Github wants to continue to lose market share at this point.

**Reply by mmisiek137** [reply]
.
